src/Peering/Peering/Peering.csproj

```diff
@@ -12,6 +12,9 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.Azure.PowerShell.AutoMapper" Version="6.2.2" />
+    <PackageReference Include="Azure.Identity" Version="1.8.0" />
+    <PackageReference Include="Azure.ResourceManager" Version="1.3.2" />
+    <PackageReference Include="Azure.ResourceManager.Peering" Version="1.1.0" />
```

Comment:
Hi @msftabdayal  currently there's no plan for Azure PowerShell to support track 2 SDK (Azure.ResourceManager.*). We offer two approaches to updating to new API version (a) generating PowerShell cmdlets directly and (b) [generating track 1 SDK and check in the code](https://eng.ms/docs/cloud-ai-platform/azure-core/azure-management-and-platforms/control-plane-bburns/azure-cli-tools-azure-cli-powershell-and-terraform/azure-cli-tools/devguide/azps/coding/generate-service-client-code). Let me know if you need more info.

Reply:
I have opened a separate PR for the powershell cmdlet generation track - will close this PR